locales/fr-FR.json

```diff
@@ -164,7 +164,7 @@
     "built_at": "Dernière compilation {0}",
     "conversations": "Conversations",
     "explore": "Explorer",
-    "favourites": "Favoris",
+    "favourites": "Aimés",
```

Comment:
I'm not fan of **Aimés** in the sidenav menu.. I like **Favoris**.

Reply:
Yet previous choices were made accordingly to popular social media practises, and that one's as much relevant.

Reply:
I'm ok to leave Favoris, WDYT?

Reply:
Alright.